simulcast.html

```diff
@@ -0,0 +1,410 @@
+        </li>
+        <li>
+           <p>Verify that each <code><a data-link-for="RTCRtpCodingParameters">rid</a></code>
+           value in <var>sendEncodings</var> is composed only of
+           alphanumeric characters (a-z, A-Z, 0-9) up to
+           a maximum of 16 characters. If one of the RIDs does not meet
+           these requirements, <a>throw</a> a <code>TypeError</code>.</p>
+        </li>
+        <li>
+           <p>If any <code><a>RTCRtpEncodingParameters</a></code>
+           dictionary in <var>sendEncodings</var> contains a
+           <a>read-only parameter</a> other than
+           <code><a data-link-for="RTCRtpCodingParameters">rid</a></code>,
+           <a>throw</a> an <code>InvalidAccessError</code>.</p>
+        </li>
+        <li>
+          <p>Let <var>maxN</var> be the maximum number of total simultaneous
+          encodings the user agent may support for this <var>kind</var>, at
+          minimum <code>1</code>.This should be an optimistic number since the
+          codec to be used is not known yet.</p>
+        </li>
+        <li>
+          <p>If the number of <code><a>RTCRtpEncodingParameters</a></code>
+          stored in <a>[[\SendEncodings]]</a> exceeds <var>maxN</var>, then trim
+          <a>[[\SendEncodings]]</a> from the tail until its length is
+          <var>maxN</var>.</p>
+        </li>
+        <li>
+          <p>If the number of <code><a>RTCRtpEncodingParameters</a></code> now
+          stored in <a>[[\SendEncodings]]</a> is <code>1</code>, then remove any
+          <code><a data-link-for="RTCRtpCodingParameters">rid</a></code>
+          member from the lone entry.</p>
+        </li>
+      </ol>
+      <p>If <var>sendEncodings</var> is set, then subsequent calls
+      to <code>createOffer</code> will be configured to send
+      multiple RTP encodings as defined in <span data-jsep=
+      "subsequent-offers initial-offers">[[!JSEP]]</span>. When
+      <code>setRemoteDescription</code> is called with a
+      corresponding remote description that is able to receive
+      multiple RTP encodings as defined in <span data-jsep=
+      "simulcast">[[!JSEP]]</span>, the
+      <code><a>RTCRtpSender</a></code> may send multiple RTP
+      encodings and the parameters retrieved via the transceiver's
+      <code>sender.getParameters()</code> will reflect the
+      encodings negotiated.</p>
+      <p>This specification
+      does not define how to configure <code>createOffer</code> to
+      receive multiple RTP encodings. However when
+      <code>setRemoteDescription</code> is called with a
+      corresponding remote description that is able to send
+      multiple RTP encodings as defined in [[!JSEP]], the
+      <code><a>RTCRtpReceiver</a></code> may receive multiple RTP
+      encodings and the parameters retrieved via the transceiver's
+      <code>receiver.getParameters()</code> will reflect the
+      encodings negotiated.</p>
```

Comment:
How would this be observable outside of receiver.getParameters()? What encoding does receiver.track display? Undefined?

The current way to get this to work is to create multiple receivers, right? SDP munging?

Reply:
Note: This text is in the current WebRTC 1.0 document.  One of the reasons to create a separate document is to better enable the existing text to be reviewed and improved upon. 

I am assuming this text is referring to a situation where an SFU is switching between simulcast streams rather than sending a single SSRC and sequence number space.  So at any given time, the SFU is only forwarding a single stream to the receiver.  However, due to reordering, loss, etc. the receiver may see packets from multiple SSRCs (and sequence number spaces) intermingled.

If the SFU is actually forwarding multiple simulcast streams to a single receiver, then all bets are off; the model for both WebRTC 1.0 and ORTC is a single receiver per track. 

Here is the note from the ORTC specification relating to this scenario:

"At the time of publication, there were two ORTC implementations supporting simulcast reception. Since neither implementation supported [RFC6051], mechanisms needed to be provided to handle intermingling of received simulcast streams due to reordering. The ORTC Lib implementation deals with this by utilizing timing heuristics as well as "hidden" receivers for each received simulcast stream, with each "hidden" receiver producing a "hidden" track. The "hidden" tracks are then mixed internally to produce a single MediaStreamTrack RTCRtpReceiver.track."

Reply:
OK thanks! That helped clarifying. I realize making all of this well defined is future effort.